app/src/main/res/layout/fragment_suggested_edits_card_item.xml

```diff
@@ -42,9 +42,12 @@
             android:id="@+id/cardItemContainer"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:addStatesFromChildren="true"
+            android:background="?attr/selectableItemBackground"
             android:clickable="true"
             android:focusable="true"
-            android:orientation="vertical">
+            android:orientation="vertical"
+            android:theme="?attr/appTouchRippleTheme">
```

Comment:
Would it be possible to just add `android:foreground="?attr/selectableItemBackground"` instead of creating a new style for only one specific ripple animation?

Reply:
Dark theme doesn't work @cooltey .

Reply:
Also I feel having a theme setting would be better than the api23 level foreground, may be for other places too? What do you feel?

Reply:
I tested on other views like list view or featured article view, the ripple animation does not work on black theme either. Would it be possible to have a separate PR of this? (and maybe a ticket for it as well?)